### PR TITLE
Fixreleasenotes

### PR DESCRIPTION
**Pull request by OsirisTerje**
Just a few more......

### CODE REVIEW

File: docs/articles/vs-test-adapter/AdapterV4-Release-Notes.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ This is a bug fix release, with the following fixes:
 * [912](https://github.com/nunit/nunit3-vs-adapter/issues/912) Explicit runs when using NUnit-filters 'cat!=FOO'
 * [914](https://github.com/nunit/nunit3-vs-adapter/issues/914) AddTestAttachment does not work anymore in VS2022
 * [918](https://github.com/nunit/nunit3-vs-adapter/issues/918) New DiscoveryMode doesn't play nicely with TestFixtureSource - Missing GenericFixture
+* [919](https://github.com/nunit/nunit3-vs-adapter/issues/919) VSTest test case filter does not work with parenthesis
```
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/AdapterV4-Release-Notes.md:11]:
```suggestion
* [919](https://github.com/nunit/nunit3-vs-adapter/issues/919) VSTest test case filter does not work with parentheses
```
Minor nitpick; may or may not be correct depending on context.

File: docs/articles/vs-test-adapter/Tips-And-Tricks.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ uid: tipsandtricks
 
 ## NUnit 3
 
-### VS Test .Runsettings configuration
+### DotNet test and VS Test .Runsettings configuration
```
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Tips-And-Tricks.md:9]: * Suggest chaining this to "dotnet run" (all lowercase) and surrounding that with backticks
* Suggest lowercase and backticks for .runsettings

**Review by SeanKilleen — APPROVED** [review]
In general looks good to me. Please see my minor comments.

Thanks!

**Review by Arhell — APPROVED** [review]

File: docs/articles/vs-test-adapter/Tips-And-Tricks.md (hunk 2 above)
Comment (OsirisTerje) [reply docs/articles/vs-test-adapter/Tips-And-Tricks.md:9]: Not sure what you mean by your first bullet line there

File: docs/articles/vs-test-adapter/Tips-And-Tricks.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ uid: tipsandtricks
 
 ## NUnit 3
 
-### VS Test .Runsettings configuration
+### DotNet test and VS Test `.runsettings` configuration
```
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Tips-And-Tricks.md:9]:
```suggestion
### `dotnet test` and VS Test `.runsettings` configuration
```

File: docs/articles/vs-test-adapter/Tips-And-Tricks.md (hunk 2 above)
Comment (SeanKilleen) [reply docs/articles/vs-test-adapter/Tips-And-Tricks.md]: I added a comment with a suggestion to clarify 👍 sorry, couldn't do it easily on mobile when I was initially reviewing.

**Pull request merged**